### PR TITLE
interactive bots: Run the links bot for anirudhjain75

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@anirudhjain75, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by anirudhjain75** [comment]
Sure in some time could you check

**Comment by femnad** [comment]
Markdown is still not 100% correctly rendered:
http://imgur.com/a/Ar0ca
Could you compare the result if the first line is a header and the three items are list items?

File: interactive-bots/links/anirudhjain75/notes.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+**Interactive bots by running the links bot**
++I have faced some issues and here are the conclusions:
```
Comment (femnad) [inline interactive-bots/links/anirudhjain75/notes.md]: Stray `+` here.

**Comment by anirudhjain75** [comment]
@femnad Ive corrected it

File: interactive-bots/links/anirudhjain75/notes.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## **Interactive bots by running the links bot**  ##
+
+**I have faced some issues and here are the conclusions:**
+
+* Always do the first step i.e, make it respond to `@links` and `@resource` in
+    followup stream and if succesfull only after that try to make it private.
```
Comment (femnad) [inline interactive-bots/links/anirudhjain75/notes.md]: Typo in succesfull

File: interactive-bots/links/anirudhjain75/notes.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## **Interactive bots by running the links bot**  ##
+
+**I have faced some issues and here are the conclusions:**
+
+* Always do the first step i.e, make it respond to `@links` and `@resource` in
+    followup stream and if succesfull only after that try to make it private. 
+* If possible try to type the code instead of copy-pasting it, It would prevent
+    your bot from not responding.
+* Ive faced the same issue and after hand-typing it, My bot worked.
```
Comment (femnad) [inline interactive-bots/links/anirudhjain75/notes.md]: Typo in I've

Comment (femnad) [inline interactive-bots/links/anirudhjain75/notes.md]: Punctuation error in `my`

File: interactive-bots/links/anirudhjain75/notes.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+## **Interactive bots by running the links bot**  ##
+
+**I have faced some issues and here are the conclusions:**
+
+* Always do the first step i.e, make it respond to `@links` and `@resource` in
+    followup stream and if succesfull only after that try to make it private. 
+* If possible try to type the code instead of copy-pasting it, It would prevent
```
Comment (femnad) [inline interactive-bots/links/anirudhjain75/notes.md]: Punctuation error in `it`

**Pull request merged**